### DISCUSSION TITLE
Turn off 12v at 0% pwm

### DISCUSSION THREAD

**Discussion by zwamdurkel** [Rem0o/FanControl.Releases#1116] [Ideas]
Some fans don't stop spinning at 0% duty cycle. I'm not sure if this is because of the fan or the motherboard.

I'm having this problem on an Asus Crosshair VI X370 with my Lian Li AL120's (PWM connected directly to the motherboard), but I have no reason to believe this won't happen with other combinations. 

One solution is using DC mode in the bios to control the fans but this also has downsides and I would rather use PWM mode. DC mode does allow the fan to fully stop since it just lowers the voltage to 0v.

My solution is as follows: make the 12v pin on the PWM header turn off when set to 0% duty cycle. This will stop the fan from spinning regardless of any other circumstances. 

Thank you for reading!

**Comment by Rem0o** [comment]
@zwamdurkel Sadly I don't have access to that. All I got is an 8 bit value that I can set 0 to 255.  If you are in PWM mode, it controls the PWm duty cycle, If you are in DC mode, it controls the 12V line.

**Comment by zwamdurkel** [comment]
I have found a workaround for my situation, So I wanted to share in case anyone else would like this functionality.
Your fans should be connected directly to the motherboard.

Step 1: set all your fans to PWM mode in the BIOS
Step 2: install ASUS AI Suite (yes I know)
Step 3: use their auto fan tuning in AI Suite
Step 4: after it is finished, turn on "Fan Stop" for every fan and set the curve to 0 as much as possible
Step 5: you can now use Fan Control to control your fans

You can now exit AI Suite and disable it from running at startup. For some reason running the auto tuning in the software enables this fan stop feature for PWM fans. There is no way to do it in the BIOS.

I saw a 50% reduction in min fan speed. From ~800rpm to ~400rpm and I now have the ability to turn the fan completely off. Make sure that Fan Control has the proper start and stop %'s. (Force mode may be necessary)

**Reply by MaximusChia** [reply]
Interesting, I have similar build. And experienced same issue when setting fans to 0%... 
What would have the Asus software that the bios dont? 
My motherboard is Strix Z790 E with Lian Li SL V1 / V2. I can also make fine tuning on bios to discover fans range %
So how you graph is? at what duty cycle fans stop? 0%?

**Reply by zwamdurkel** [reply]
I have switched to an ASRock Taichi Z790 for unrelated reasons and it allows fan stop in the bios, so I can no longer answer this question.
